### ISSUE TITLE
Accordion: Use new focus style

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystem#426]
The accordion should have a focus style as the following image with `border_radius-interactive`.

![Image](https://github.com/digdir/designsystem/assets/46874830/80a1ae9d-4074-411b-8996-608ed014ba6d)
